anyway/parsers/infographics_data_cache_updater.py

```diff
@@ -112,7 +144,7 @@ def get_cache_info():
         .count()
     )
     db.session.commit()
-    return f"num items in cache: {cache_items3},{cache_items5}, temp table: {tmp_items3},{tmp_items5}, accident flashes:{num_acc_flash_items}, flashes processed:{num_acc_suburban_flash_items}"
+    return f"num items in cache: {cache_items}, temp table: {tmp_items}, accident flashes:{num_acc_flash_items}, flashes processed:{num_acc_suburban_flash_items}"
```

Comment:
Why are we returning this result? are we parsing it later?

Reply:
It is not parsed, just printed.